audio/tsunami/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
+
+github.setup            momentarylapse tsunami 23a4621c7f2d812e06f0cf6d3ac9553421666ccf
```

Comment:
as usual, please do not use git hashes instead you should use a released tag used by upstream.

Reply:
@reneeotten This is not possible, release is broken for macOS. Please follow the issue I refer: https://github.com/momentarylapse/tsunami/issues/12

Reply:
Do we have a resolution on this?

Reply:
Well I do not see a problem to switch to release once it is there. I do not pick commits just for fun or to annoy someone :) And I do move to released versions once it works, I did that with other ports.